frontend/src/pages/budgetLines/list/BudgetLineItemList.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { useSearchParams } from "react-router-dom";
+import _ from "lodash";
+import App from "../../../App";
+import { useGetBudgetLineItemsQuery, useGetCansQuery, useGetAgreementsQuery } from "../../../api/opsAPI";
+import Breadcrumb from "../../../components/UI/Header/Breadcrumb";
+import Alert from "../../../components/UI/Alert";
+import TablePageLayout from "../../../components/UI/Layouts/TablePageLayout";
+import AllBudgetLinesTable from "../../../components/UI/AllBudgetLinesTable";
+
+/**
+ * Page for the Budget Line Item List.
+ * @returns {React.JSX.Element} - The component JSX.
+ */
+export const BudgetLineItemList = () => {
+    const [searchParams] = useSearchParams();
+    const isAlertActive = useSelector((state) => state.alert.isActive);
+    const [filters, setFilters] = React.useState({});
+
+    const {
```

Comment:
this works for now but hoping there is a better way to not make 3 api calls and pull in all BLIs, Agreements, and CANs and merge together to get the Agreement name and CAN number.